Fork of codesandbox doesn't compile

**What you were expecting:**
Fork https://codesandbox.io/s/github/marmelab/react-admin/tree/master/examples/simple, make changes, code should compile.

**What happened instead:**
Code fails to compile with this error:

```
ERROR in ./src/polyfills.ts
Module not found: Error: Can't resolve 'proxy-polyfill/proxy.min.js' in '/sandbox/src'
 @ ./src/polyfills.ts 5:0-38
 @ multi (webpack)-dev-server/client?http://127.0.0.1:8080 (webpack)/hot/dev-server.js ./src/polyfills.ts
```

**Steps to reproduce:**
Fork https://codesandbox.io/s/github/marmelab/react-admin/tree/master/examples/simple
open any js file and add `console.log("test")`
see compile error in terminal

**Related code:**

https://codesandbox.io/s/elegant-dust-pdqkl?file=/src/comments/CommentEdit.tsx

This is preventing me from creating a simple example of my issue in https://github.com/marmelab/react-admin/issues/6751

--- a/examples/simple/package.json
+++ b/examples/simple/package.json
@@ -26,6 +26,7 @@
         "html-loader": "~1.3.0",
         "html-webpack-plugin": "~4.3.0",
         "ignore-not-found-export-plugin": "^1.0.1",
+        "proxy-polyfill": "^0.3.0",
         "react-app-polyfill": "^1.0.4",
         "style-loader": "~1.2.1",
         "typescript": "^4.4.0",

